.gitignore

```diff
@@ -47,4 +47,5 @@ tmp/
 target/
 
 /build/
-lumo-cache/
\ No newline at end of file
+lumo-cache/
+third_party/
```

Comment:
this is not needed anymore, better to keep the diff clean and not touch this file